src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -78,10 +77,6 @@ class SparqlQleverVisitor : public SparqlAutomaticVisitor {
   void setPrefixMapManually(PrefixMap map) { _prefixMap = std::move(map); }
 
  private:
-  // For the unit tests
-  PrefixMap& prefixMap() { return _prefixMap; }
-  FRIEND_TEST(SparqlParser, Prefix);
-
```

Comment:
For the reasons stated above I would like to keep these members as I suspect we will need them.

Reply:
So you want keep both the member for testing and the public member?

Reply:
Together with my comments from above my opinion about the `_prefixMap`being the most important thing (also to test) for the future should be clear.
It might be, that the non-const access here is not needed anymore, because we typically pass in an initial prefix map using the `ParserAndVisitor` class and NOT via this member. In this case it can be removed (I am not a gread friend of `friend and FRIEND_TEST` in general).
In this case, don't forget to remove the `include<gtest>` if this was the only friend.